### ISSUE TITLE
is docummentation lost?

### ISSUE THREAD

**Issue by alfonz19** [statefulj/statefulj#35]
When I try to reach documentation, it yields info, that domain is for sale... Can you update the links / webpage?
Thanks!

**Comment by nhstevehz1** [comment]
Yes, please make the documentation available.  StatefulJ is exactly what I need for one of my projects.
